mandatory/3-playing-computer.js

```diff
@@ -17,6 +17,7 @@
 
 let x = 2;
 let a = 6;
+let b = 4;
```

Comment:
Yes this lets the code execute by resolving the reference error. The alternative would be to remove the `console.log(b);` line, since variable `b` is not used anywhere in the code. But what you've done is sensible too - and the question doesn't tell you what the code is supposed to do, so your approach also makes sense.